### DISCUSSION TITLE
RuntimeError: product name has been set to "Pyarmor"

### DISCUSSION THREAD

**Discussion by yahav6893** [dashingsoft/pyarmor#1045] [Q&A]
I upgraded pyarmor to 8.0.21 so I could update the license to fit the 8.0+ version.

Getting this error message:
![image](https://user-images.githubusercontent.com/50764241/228524446-6e01bd1d-9e50-4c03-aa03-cdd1837178b5.png)

Running on ubuntu 18.04.

Thanks ahead

**Comment by jondy** [comment]
It's you set product name to `Pyarmor` when purchasing license.

And product name can't be changed according to Pyarmor EULA.
